Specs/Core/HeadingPitchRollSpec.js

```diff
@@ -51,6 +51,32 @@ defineSuite([
         }
     });
 
+    it('conversion from direct quaternion', function() {
+        var testingTab = [
+            [0, 0, 0],
+            [90 * deg2rad, 0, 0],
```

Comment:
Maybe this file was already like this, but why not use the conversion functions in `Cesium.Math`?

Reply:
In the old PR, I used [this variable](https://github.com/kaktus40/cesium/blob/db16f8f9f3c5da35225326d7a24369ed51c60aec/Specs/Core/HeadingPitchRollSpec.js#L12), which  is also a shortcut to CesiumMath.RADIANS_PER_DEGREE, for its concision.